### PR TITLE
Use Ctrl-H to switch to Find-Replace dialog from the other tabs

### LINKED ISSUE #10019: CTRL+F should upgrade to CTRL+H when pressed
## Currently
When on the CTRL+F window, pressing CTRL+H does nothing (or something unexpected).

## Expected
What _should_ intuitively happen like in most (if not all) other editors:

When on the CTRL+F window, **pressing CTRL+H should "upgrade" to what should normally show when pressing CTRL+H**.

## Debug
Notepad++ v7.9.5   (32-bit)
Build time : Mar 21 2021 - 02:09:07
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 2009
OS Build : 19043.1055
Current ANSI codepage : 1252
Plugins : JSLintNpp.dll JSMinNPP.dll mimeTools.dll NppConverter.dll NppExport.dll

### PR DESCRIPTION
**Pull request by donho**
Add find/replace/... shortcuts detection capacity in Find Replace dialog.

Fix #10019

### CODE REVIEW

File: PowerEditor/src/WinControls/shortcut/shortcut.cpp [hunk 1]
```diff
@@ -582,11 +581,13 @@ void Accelerator::updateShortcuts()
...
+	size_t nbFindReplaceAcc = findReplaceAcc.size();
+	if (nbFindReplaceAcc)
 	{
-		ACCEL *tmpFindAccelArray = new ACCEL[1];
-		tmpFindAccelArray[0] = *pSearchFindAccel;
-		_hFindAccTab = ::CreateAcceleratorTable(tmpFindAccelArray, 1);
+		ACCEL* tmpFindAccelArray = new ACCEL[nbFindReplaceAcc];
+		for (size_t i = 0; i < nbFindReplaceAcc; ++i)
+			tmpFindAccelArray[i] = findReplaceAcc[i];
+		_hFindAccTab = ::CreateAcceleratorTable(tmpFindAccelArray, nbFindReplaceAcc);
```
Comment (mere-human) [inline PowerEditor/src/WinControls/shortcut/shortcut.cpp:590]: warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of data

File: PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp [hunk 2]
```diff
@@ -1207,7 +1207,19 @@ INT_PTR CALLBACK FindReplaceDlg::run_dlgProc(UINT message, WPARAM wParam, LPARAM
 				return TRUE;
 
 				case IDM_SEARCH_FIND:
-					goToCenter();
```
Comment (mere-human) [inline PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp:1210]: By the way, how is it managed by the system?
Is there a way to "find" the Find dialog now?

**Comment by mere-human** [comment]
I verified it locally.
It actually works and allows you to switch from Find (_Ctrl+F_) to Replace (_Ctrl+H_) which is good.
It kind of solves the problem.

However, as for me, _Ctrl+Tab_ would still be more logical.
Since, I personally don't remember all the shortcuts.

Also, there is no shortcut for _Find in Projects_ so there is no way to select it using a keyboard.
image

image

File: PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp (hunk 2 above)
Comment (donho) [reply PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp:1210]: I tested roughly. If Find dialog is off screen but Notepad++ is on the screen, Ctrl-F will bring back Find dialog in the main frame zone of Notepad++ with or without dialog being focused.

**Comment by donho** [comment]
> However, as for me, Ctrl+Tab would still be more logical. Since, I personally don't remember all the shortcuts.

Ctrl-F & Ctrl-H are much more intuitive, that's why such request exists. For the Ctrl-Shift-F (Find in files) & Ctrl-M (Mark), users will check the shorcuts if needed. 

These shortcuts are already used, so it makes more sense to me to add them. Not mentioning without the lines and lines added code which could bring bugs/regression.

File: PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp (hunk 2 above)
Comment (mere-human) [reply PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp:1210]: Indeed, I can confirm.

**Pull request closed without merging**